util.go

```diff
@@ -28,6 +28,18 @@ func (b *Bot) deferDebug() {
 	}
 }
 
+func (b *Bot) runHandler(handler func()) {
+	f := func() {
+		defer b.deferDebug()
```

Comment:
Note that without this PR `deferDebug` is deferred only if b.reporter is nil, but it seems that it should be always deferred, since without it no one handles panics in handlers, whereas Settings.Reporter says that it should be called on panic.

Reply:
Good catch!